testsuite/smoke-test/src/storage.rs

```diff
@@ -284,9 +284,11 @@ fn transfer_and_reconfig(
         if random::<u16>() % 10 == 0 {
             println!(
                 "Changing diem version to {}: {:?}",
-                transferred,
-                client
-                    .change_diem_version(&["change_diem_version", &transferred.to_string()], true)
+                transferred + 1.0,
+                client.change_diem_version(
+                    &["change_diem_version", &(transferred + 1.0).to_string()],
```

Comment:
seems wrong we have to change this to pass test

Reply:
do you mean the test is wrong?

Reply:
no, if the test passes today, it should pass without changes.